test/integration/targets/any_errors_fatal/80981.yml

```diff
@@ -0,0 +1,17 @@
+- hosts: testhost,testhost2
+  gather_facts: false
+  any_errors_fatal: true
+  tasks:
+    - block:
+        - fail:
+          when: inventory_hostname == "testhost"
+        - name: any_errors_fatal fails all hosts when any of them fails
```

Comment:
This would be a change in behavior, after `rescue` both would show up as rescued which might be confusing because only one actually failed explicitly in a play while others where failed by `any_errors_fatal`.

Reply:
still, much better than the existing behavior